src/libUtils/UpgradeManager.cpp

```diff
@@ -394,37 +394,46 @@ bool UpgradeManager::ReplaceNode(Mediator& mediator)
     LOG_MARKER();
 
     if (LOOKUP_NODE_MODE)
-    {
-        LOG_GENERAL(WARNING,
-                    "For LookUp node, temporarily disable upgrading protocol.");
-        return true;
-    }
-
-    if (DirectoryService::IDLE == mediator.m_ds->m_mode)
     {
         LOG_GENERAL(INFO,
-                    "Shard node, upgrade after "
-                        << TERMINATION_COUNTDOWN_IN_SECONDS << " seconds...");
-        this_thread::sleep_for(
-            chrono::seconds(TERMINATION_COUNTDOWN_IN_SECONDS));
-    }
-    else if (DirectoryService::BACKUP_DS == mediator.m_ds->m_mode)
-    {
-        LOG_GENERAL(INFO,
-                    "DS backup node, upgrade after "
-                        << TERMINATION_COUNTDOWN_IN_SECONDS + 1
+                    "Lookup node, upgrade after "
+                        << TERMINATION_COUNTDOWN_IN_SECONDS + 3
                         << " seconds...");
         this_thread::sleep_for(
-            chrono::seconds(TERMINATION_COUNTDOWN_IN_SECONDS + 1));
+            chrono::seconds(TERMINATION_COUNTDOWN_IN_SECONDS + 3));
```

Comment:
Why use + 1, + 2, + 3 here? How does the delay is determined?

Reply:
Ok, I will define some meaningful variables for them.